1906-10-05.xml

```diff
@@ -0,0 +1,7096 @@
+                    <p>The Cairo branch of the Comptoir National d'Escompte, Paris, to be
+                        inaugurated on the 15th inst., will be temporarily housed in the Maison
+                        Manusardi. Emad el Dio Avenne, near the Hotel d'Angleterre. </p>
+                    </div>
+                    <div type="item">
+                    <head>National University.</head>
+                    <p>The subscription towards the foundation of a National University has reached
+                        LE. 2,100.1. It is feared in some quarters that this scheme of a National
+                        University will share the same fate as that drawn up by several Khedivial
+                        princes and Egyptian notables at the time of the Mohamed Ali centenary. </p>
+                    </div>
+                    <div type="item">
+                    <head>Suez Canal</head>
+                    <p>On October 3, 6 vessels, 2 northward and 4; southward bound, transited the
+                        Canal and paid Fr. 177,712 50 for transit and passenger toll. Of these
+                        vessels, 3 were British, 2 German and Austrian. The total receipts since
+                        October 1 are Fr. 639,119.92, paid by 25 vessels, of which 9 were northward
+                        and 16 southward bound.</p>
+                </div>
+                    <div type="item">
+                    <head>Slave Trade.</head>
+                    <p>Mohamed Naeser, the notorious slave dealer, has at last been caught, after
+                        having eluded the vigilance of be authorities for eight years. Mohamel, who
+                        is the head of the El Parayat | tribe, is famous for the pitobed battle he
+                        fought with the police on September 28, 1905, where | ten of his band were
+                        arrested and condemned to 15 years' hard labor. Lately the band were
+                        surprized with 12 slaves, whom the patrol, however, were unable to liberate,
+                        owing to the resistance shown by the Bedonin. </p>
+                </div>
+                    <div type="item">
+                    <head>New Employment Registry.</head>
+                    <p>The British Chamber of Commerce gives notice that it has commenced an
+                        employment register, which may be consulted by members requiring clerical or
+                        other assistance. Applications will be received, free of charge, from clerks
+                        and others wishing to obtain posts in Egypt. Address, Secretary, British
+                        Chamber of Commerce, Alexandria. [Advt.]</p>
+                    </div>
+                    <cb/>
+                        </div>
+                
+                <div type="section">
+                    <head>ITALIAN DIPLOMATIC SERVICE.</head>
+                    </div>
```

Comment:
don't end the div with only a title. You can have two lines of `<head>`. Just delete 2203 and 2204.